sklearn/utils/estimator_checks.py

```diff
@@ -597,7 +598,7 @@ def check_fit2d_1sample(name, estimator_orig):
 
 @ignore_warnings
 def check_fit2d_1feature(name, estimator_orig):
-    # check by fitting a 2d array and prediting with a 1d array
+    # check by fitting a 2d array with only one feature
     rnd = np.random.RandomState(0)
     X = 3 * rnd.uniform(size=(10, 1))
     y = X[:, 0].astype(np.int)
```

Comment:
For the specific purpose of this test, i.e accepting 1d X or returning informative error, we can set `k='all'` for `SelectKBest`, `sample_fraction=1` for `RandomizedLogisticRegression` and `residual_threshold=0.5` for `RANSACRegressor`.